src/test/java/uk/ac/cam/cl/dtg/isaac/api/managers/GameManagerTest.java

```diff
@@ -92,4 +96,81 @@ public void getNextQuestionsForFilter_appliesExclusionFilterForDeprecatedQuestio
     assertEquals(deprecatedFilter.getOperator(), Constants.BooleanOperator.NOT);
     assertEquals(deprecatedFilter.getValues(), Collections.singletonList("true"));
   }
+
+  @Test
+  public void generateRandomQuestions_returnsCorrectNumberOfQuestions() throws ContentManagerException {
+
+    // Arrange
+    int limit = 5;
+
+    var gameManager = new GameManager(
```

Comment:
You seem to always be initialising `gameManager` in the same way, so you can move these initialisations to the existing `setUp` method.